### DISCUSSION TITLE
How to use image mounter for iOS 17+ on linux

### DISCUSSION THREAD

**Discussion by ghost** [doronz88/pymobiledevice3#1069] [Q&A] [answered]
Hello, I'm a little lost so I'll explain the context. I have a non-jailbroken iPhone 11, I injected frida-gadget into an ipa and want to use tools like "frida-ps" from my linux computer. Issue: `Failed to enumerate apps: This feature requires mounting an iOS developer disk image; run Xcode briefly or use ideviceimagemounter to mount one manually.` So I inquired but no iOS 17 developer image is online because the method has changed (I think). So how do I get this image? I have a Mac that I can use occasionally, if ever it was possible to "dump" the iOS 17 developer image. Thanks

**Comment by doronz88** [comment, marked as answer]
```shell
pymobiledevice3 mounter auto-mount
```

**Reply by ghost** [reply]
Thanks !
